pkg/infoschema/tables.go

```diff
@@ -1663,6 +1666,22 @@ var tableKeywords = []columnInfo{
 	{name: "RESERVED", tp: mysql.TypeLong, size: 11},
 }
 
+var tableTiDBParamsCols = []columnInfo{
+	{name: "COMPONENT", tp: mysql.TypeVarchar, size: 64, flag: mysql.NotNullFlag},
+	{name: "TYPE", tp: mysql.TypeVarchar, size: 64, flag: mysql.NotNullFlag},
+	{name: "NAME", tp: mysql.TypeVarchar, size: 256, flag: mysql.NotNullFlag},
+	{name: "INSTANCE", tp: mysql.TypeVarchar, size: 64},
+	{name: "SCOPE", tp: mysql.TypeVarchar, size: 64, flag: mysql.NotNullFlag},
+	{name: "VALUE", tp: mysql.TypeLongBlob, size: types.UnspecifiedLength, flag: mysql.NotNullFlag},
+	{name: "DEFAULT_VALUE", tp: mysql.TypeVarchar, size: 64},
```

Comment:
Why isn't the `VALUE` the same type as the `DEFAULT_VALUE`?

Reply:
`DEFAULT_VALUE` is shorter than `VALUE `, such as `tidb_config`.  
Because `DEFAULT_VALUE` is NULL in most cases.